es/politica-de-privacidad.md

```diff
@@ -0,0 +1,62 @@
+```
```

Comment:
Two changes needed:

1. remove the ``` - these don't belong around the YAML headers in a markdown file
2. Please add `original: privacy-policy` to the metadata so that the translation is properly linked